executor.lua

```diff
@@ -31,9 +31,19 @@ missions.start = function(pos, player)
 		description = meta:get_string("description") or ""
 	}
 
+	local nointeract = meta:get_int("nointeract")
+
+	if nointeract then
+		local privs = minetest.get_player_privs(playername)
+		privs.interact = false
```

Comment:
Needs to be nil, or this will grant rather than revoke.

Same thing for `privs.mission_nointeract = false` below.